ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/event-handlers.md

```diff
@@ -46,6 +46,9 @@ interface Management1
 | CONFIG_VERSIONING | bash | <code>FN=/mnt/flash/startup-config; LFN="`ls -1 $FN.*-* \| tail -n 1`"; if [ -z "$LFN" -o -n "`diff -I 'last modified' $FN $LFN`" ]; then cp $FN $FN.`date +%Y%m%d-%H%M%S`; ls -1r $FN.*-* \| tail -n +11 \| xargs -I % rm %; fi</code> | on-startup-config |
 | evpn-blacklist-recovery | bash | <code>FastCli -p 15 -c "clear bgp evpn host-flap"</code> | on-logging |
 | trigger-on-boot | bash | <code>echo "on-boot"</code> | on-boot |
+| trigger-on-maintenance1 | bash | <code>echo "on-maintenance"</code> | on-maintenance |
+| trigger-on-maintenance2 | bash | <code>echo "on-maintenance"</code> | on-maintenance |
+| trigger-on-maintenance3 | bash | <code>echo "on-maintenance"</code> | on-maintenance |
```

Comment:
Seems like we should add some more info here about the trigger. There is no way to see which interface or which unit etc. We can do it in a later PR.

Reply:
Sure

Reply:
There is one more issue for event-handlers- https://github.com/aristanetworks/avd/issues/3267
Will work on it once this one is merged. (we can do this documentation updates in that PR, if its fine for you)

Reply:
Sure.